wikimedia/wikipedia-ios
# Locked time zone for tests

**Phabricator:** N/A

### Notes
`testEditMilestoneHundredOnUserTalkPage()` in the default test suite would fail if the test system was set to a time zone far enough east. This is due to the fact that it compares against the string literal `"7/16/21"`, but uses a date formatter that depends on the current system calendar/time zone/etc

This PR addresses this issue by locking the time zone on the test system to `America/New_York`, thereby eliminating the system time zone as a dependency of the tests

### Test Steps
1. Set the system the tests will be run upon to `Europe/Amsterdam` (or other Central European time zone)
2. Run the default `Wikipedia` test suite
3. Observe that all the tests (including `testEditMilestoneHundredOnUserTalkPage()`) pass
